docs/content/platform/blocks/ayrshare/post_to_tiktok.md

```
@@ -0,0 +1,52 @@
+| random_media_url | Whether to generate random media | bool | No |
+| notes | Additional notes for the post | str | No |
+| auto_add_music | Whether to automatically add recommended music to the post. If you set this field to true, you can change the music later in the TikTok app. | bool | No |
+| disable_duet | Disable duets on published video (video only) | bool | No |
+| disable_stitch | Disable stitch on published video (video only) | bool | No |
+| is_ai_generated | If you enable the toggle, your video will be labeled as “Creator labeled as AI-generated” once posted and can’t be changed. The “Creator labeled as AI-generated” label indicates that the content was completely AI-generated or significantly edited with AI. | bool | No |
+| is_branded_content | Whether to enable the Branded Content toggle. If this field is set to true, the video will be labeled as Branded Content, indicating you are in a paid partnership with a brand. A “Paid partnership” label will be attached to the video. | bool | No |
+| is_brand_organic | Whether to enable the Brand Organic Content toggle. If this field is set to true, the video will be labeled as Brand Organic Content, indicating you are promoting yourself or your own business. A “Promotional content” label will be attached to the video. | bool | No |
+| image_cover_index | Index of image to use as cover (0-based, image posts only) | int | No |
+| title | Title for image posts | str | No |
+| thumbnail_offset | Video thumbnail frame offset in milliseconds (video only) | int | No |
+| visibility | Post visibility: 'public', 'private', 'followers', or 'friends' | "public" | "private" | "followers" | No |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)